react/features/recording/components/LiveStream/BroadcastList.web.js

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import { translate } from '../../../base/i18n';
+
+import BroadcastListItem from './BroadcastListItem';
+
+/**
+ * A React Component for display information for a live stream.
+ *
+ * @extends Component
+ */
+class BroadcastList extends Component {
+    /**
+     * {@code BroadcastList} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        broadcasts: PropTypes.array,
+        onClick: PropTypes.func,
+        t: PropTypes.func
+    };
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        const broadcasts = this.props.broadcasts.map(broadcast =>
+            this._renderBroadcastItem(broadcast));
+
+        return (
+            <div
+                className = 'live-stream-item'
+                onClick = { this._onClick }>
+                { broadcasts.length
+                    ? broadcasts
+                    : this.props.t('liveStreaming.noBroadcasts') }
+            </div>
+        );
+    }
+
+    /**
+     * Renders a react element.
+     *
+     * @param {Object} broadcast - An object with information about the
+     * broadcast.
+     * @private
+     * @returns {ReactElement}
+     */
+    _renderBroadcastItem(broadcast) {
```

Comment:
i see a mix of `broadcastItem` and `broadcastListItem` is it worth making all of them consistent?